nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -61,6 +73,7 @@ let
       #Let's hope 15M is enough
       dd bs=2048 count=7680 if=/dev/zero of="$out"
```

Comment:
If i may, increasing this to 100MB or 512MB would be ideal, on install the EFI Parition needs to be 512MB to allow for multiple initrd's etc and also its more the defacto see: http://en.wikipedia.org/wiki/Unified_Extensible_Firmware_Interface#Linux

Will also allow people to put other images on to the USB stick should people want to dual boot it. Just an opinion.